onnxscript/ir/passes/common/constant_manipulation.py

```diff
@@ -23,11 +23,14 @@ class LiftConstantsToInitializersPass(ir.passes.InPlacePass):
     Attributes:
         lift_all_constants: Whether to lift all Constant nodes, including those that does not contain a tensor attribute (e.g. with value_ints etc.)
             Default to False, where only Constants with the ``value`` attribute are lifted.
+        size_limit: The minimum size of the tensor to be lifted. If the tensor contains
+            less than this number of elements, it will not be lifted.
```

Comment:
nit: confusing wording, maybe
```suggestion
            number of elements less than size_limit, it will not be lifted.
```